.shed.yml

```diff
@@ -0,0 +1,14 @@
+owner: creason
+remote_repository_url: https://github.com/ohsu-comp-bio/unmicst
+categories: 
+  - "Imaging"
+repositories:
+  unmicst:
+    description: "Images can be preprocessed by inferring nuclei contours via a pretrained UNet model."
+    include: 
```

Comment:
Better remove or use exclude, otherwise you need to update this list and our test will not tell you when the include list is missing file to upload to the TS.